tests/baselines/reference/noImplicitAnyStringIndexerOnObject.js

```diff
@@ -1,7 +1,43 @@
 //// [noImplicitAnyStringIndexerOnObject.ts]
-var x = {}["hello"];
-var y: string = { '': 'foo' }[''];
+var a = {}["hello"];
+var b: string = { '': 'foo' }[''];
+
+// Should give suggestion 'c.get'
+var c = {
+  get: (key: string) => 'foobar'
+};
+c['hello'];
+
+// Should give suggestion 'd.set'
+var d = {
+  set: (key: string) => 'foobar'
+};
+d['hello'];
+
+// Should give suggestion 'e.get or e.set'
```

Comment:
I would try to avoid (and remove) these comments regarding error messages in general. The baselines should communicate that on their own.